File: CURSO PHP POO/CURSO POO PHP/Aula11/Bolsista.php
<?php
require_once 'Aluno.php';
class Bolsista extends Aluno{
    private $bolsa;
    
    public function renovarBolsa() {
        echo "<p>Bolsa Renovada</p>";
    }
    
    public function pagarMensalidade() {
        echo "<p>$this->nome é bolsista! Então paga com desconto!</p>".$this->nome;
    }
    
    function getBolsa() {
        return $this->bolsa;
    }

    function setBolsa($bolsa) {
        $this->bolsa = $bolsa;
    }


}

File: EXERCÍCIOS JAVA E JAVA POO/Aula11_POO/src/aula11_poo/Bolsista.java
package aula11_poo;
public class Bolsista extends Aluno {
    private float bolsa;
    
    public void renovarBolsa(){
        System.out.println("Renovando bolsa de "+this.nome);
    }
    @Override//@Override : Sobreposição a outro método
    public void pagarMensalidade(){
        System.out.println(this.nome+" é bolsista pagamento facilitado! ");
    }

    public float getBolsa() {
        return bolsa;
    }

    public void setBolsa(float bolsa) {
        this.bolsa = bolsa;
    }
    
}
